pkg/sql/logictest/testdata/logic_test/udf

```diff
@@ -174,6 +174,13 @@ FROM pg_catalog.pg_proc WHERE proname IN ('proc_f', 'proc_f_2');
 100119  proc_f    105  1546506610  14  true   true   false  i  2  25  25 20  {i,i}  {"",b}  SELECT 'hello';
 100121  proc_f_2  120  1546506610  14  false  false  false  v  1  25  25     {i}    NULL    SELECT 'hello';
 
+# Ensure that the pg_proc virtual index works properly.
+
+query TT
+SELECT oid, proname FROM pg_proc WHERE oid = 'sc.proc_f_2'::regproc
```

Comment:
nit: force the index with `pg_proc@pg_proc_oid_idx`

Reply:
You can't force virtual indexes - the optimizer gives an error.